### ISSUE TITLE
Can we use some of the optimization functions on other models unrelated to SD and diffusion ?

### ISSUE THREAD

**Issue by oxysoft** [chengzeyi/stable-fast#120]
I see many functions that seem general enough. If we have a depth estimation model in the artistic video pipeline for example, is it possible to introduce them to various optimizations and compiler features of this repo?

Cheers

**Comment by chengzeyi** [comment]
@oxysoft Yes, you can adopt some fine-tuned kernels and graph-capture mechanisms from `stable-fast` if they can help.
